ideascube/models.py

```diff
@@ -244,6 +244,14 @@ def index_strings(self):
         ('female', _('Female')),
     )
 
+    DISABILITY_CHOICES = (
+        ('visual', _('Visual')),
+        ('auditive', _('Auditive')),
+        ('physical', _('Physical')),
+        ('cognitive', _('Cognitive')),
+        ('mental', _('Mental')),
```

Comment:
This is a specific need for this project. In BSF' context, Yohan's list in #608 is exhaustive enough. If we want to be sure not to leave anyone on the side, adding a "other" might be an acceptable alternative.